README.md

```diff
@@ -64,9 +64,18 @@ end
 TODO
 ```
 
-## Documentation
+## Documentation - APIS
 
-TODO (e.g. link to wiki page)
+
+Following table shows the API endpoints of the Policy Manager.
+
+| Action | HTTP Method | Endpoint |
+| --------------- | ------- | -------------------------------------------- |
+| Create a Policy | `PUT` |`/api/policymngr/v1/policy-descriptor`|
```

Comment:
@jbonnet since the policies are going to be created by the UI (one by one via the policy editor) there is no sense in creating them i a massive call. Each create call refers to one policy.

Reply:
Right... the difference is for `PUT`ing you need the ID, while for `POST`ing the ID is created

Reply:
@jbonnet thanks for clarifying . I switched to POST :-)